cmake/stm32/common.cmake

```diff
@@ -205,6 +210,8 @@ function(stm32_get_memory_info)
         set(FLASH "32K")
     elseif(SIZE_CODE STREQUAL "8")
         set(FLASH "64K")
+    elseif(SIZE_CODE STREQUAL "A")

```

Comment:
I think this is not completely correct. all size codes are incorrect for MP1 family. It must be managed explicitly and not just removing a warning.

Reply:
There's no flash on the MP1 family. The only size code is "A" and I though that's what it meant. Don't you agree?
I did not put that there just to remove a warning. 

To me, this part just don't apply for the MP1 family.

Reply:
in STM32G473RE the E is the size code. 
Please also have a look at datasheet. From my point of view, MP1 has no such thing as a size_code.
maybe you should do same as L1 family (see `stm32l1_get_memory_info`) to return always 0. 
All these information makes me think a modification is needed.
![image](https://user-images.githubusercontent.com/4628382/154802746-775a9507-111d-47eb-a3b7-79fb079f327d.png)